e2e-tests/src/synthetic_network.rs

```diff
@@ -22,3 +33,56 @@ pub async fn set_out_latency(milliseconds: Milliseconds, synthetic_url: String)
         .await
         .expect("unable to commit network configuration");
 }
+
+pub async fn wait_for_further_finalized_blocks(
```

Comment:
Why `pub` and why here? :)

Reply:
I understand this file contains helper  (utils) functions for actual test case code. WIthout `pub` you won't be able to use it in  `src/test/high_latency.rs` file.`

Reply:
RIght, no need for `pub`, removed. In this PR I don't want to modify `aleph-client`'s API, so let's just remove `pub` for now.